lib/JSON/Schema/Fit.pm

```diff
@@ -86,6 +86,19 @@ Default: 1
 
 sub round_numbers { return _attr('round_numbers', @_); }
 
+=attr limits
```

Comment:
Something like 'crop_numbers' will be better

Reply:
Hi. Curiously, I replied to github email, but I do not see here the thread. 
Some libraries use clamp for this kind of operation: https://docs.unity3d.com/ScriptReference/Mathf.Clamp.html
What do you prefer, `crop` or `clamp`?

Reply:
Hi! Sorry for the delay.
Clamp is better, i just hadn't found this term.
I'll fix it myself